src/API/WorkController.ts

```diff
@@ -26,9 +26,11 @@ export class WorkController {
     return this.collection.findOne({ id }, { fields: { _id: false } })
   }
 
-  async getByPublicKey(publicKey: string): Promise<any> {
+  async getByPublicKey(publicKey?: string): Promise<any> {
```

Comment:
`getByPublicKey` no longer represents what this method does. Maybe `getByFilters`?